### ISSUE TITLE
can this run on ARM architecture and more specifically on TK1 kit ?

### ISSUE THREAD

**Issue by aliikram97** [Celebrandil/CudaSift#79]
i tried to run with msse2 but it due to different architecture did not work... i removed msse and got the following error /....

/home/ubuntu/aliexperiments/CudaSift/cudaSiftD.cu(1319): error: identifier "__any_sync" is undefined

1 error detected in the compilation of "/tmp/tmpxft_000028fb_00000000-6_cudaSiftH.cpp1.ii".
CMake Error at cudasift_generated_cudaSiftH.cu.o.cmake:279 (message):
  Error generating file
  /home/ubuntu/aliexperiments/CudaSift/build/CMakeFiles/cudasift.dir//./cudasift_generated_cudaSiftH.cu.o

make[2]: *** [CMakeFiles/cudasift.dir/cudasift_generated_cudaSiftH.cu.o] Error 1
make[1]: *** [CMakeFiles/cudasift.dir/all] Error 2
make: *** [all] Error 2

**Comment by Celebrandil** [comment]
I managed to run it on a Jetson TK1 before. It could be that you don't use CUDA 9.0 or later, since __any_sync used to be called __any in earlier versions.

**Comment by aliikram97** [comment]
Yes that was the case i down graded these functions and it worked but what
is confusing me is that it takes alot of time to compute and doesn't
perform very good when tested on thermal images. If some assistance in
understanding the code file which.eill allow to change the parameters of
the function. Also do you use RANSAC for homography matrix ?
